ads/google/a4a/utils.js

```diff
@@ -707,6 +708,23 @@ export function extractAmpAnalyticsConfig(a4a, responseHeaders) {
     // Security review needed here.
     config['requests'] = requests;
     config['triggers']['continuousVisible']['request'] = Object.keys(requests);
+
+    // Add begin to render requests
+    const btrUrls = analyticsConfig['btrUrl'];
+    if (Array.isArray(btrUrls) && btrUrls.length) {
+      const btrRequests = dict();
+      for (let idx = 1; idx <= btrUrls.length; idx++) {
```

Comment:
nit: I would find this very slightly clearer if you iterated over `btrUrls` in the ordinary (zero indexed) fashion, and then referred to `btrRequests['btr${idx+1}']`

Reply:
I agree. Switched to zero index.